test/plugins/builds.test.js

```diff
@@ -2096,23 +2099,234 @@ describe('build plugin test', () => {
                         { src: 'd', dest: 'c', join: true }
                     ];
 
-                    buildMock.parentBuilds = {
+                    const buildC = {
+                        jobId: 3,
+                        status: 'CREATED',
+                        parentBuilds: {
+                            123: {
+                                eventId: '8888', jobs: { a: null, d: 5555 }
+                            }
+                        }
+                    };
+                    const updatedBuildC = Object.assign(buildC, {
+                        parentBuilds: {
```

Comment:
is the logic for updating `parentBuilds` tested elsewhere ? since we are dealing with stubs, want to make sure that it's handled properly.